index.html

```diff
@@ -16,12 +16,61 @@
 
 <body>
 
```

Comment:
What happened to `<div class="container">` ? You still have the closing tag on line 74